### PR TITLE
*: refactor a function name

### PR DESCRIPTION
**Pull request by zhexuany**
This changes makes function much more readable. Original `SupportRequestType` may confused people  that this function actually return a RequestType. By renaming it as `IsRequestTypeSupported`, confusion will go away.

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tidb/3263)

### CODE REVIEW

**Comment by zhexuany** [comment]
@shenli @hanfei1991 PTAL.

**Comment by ngaut** [comment]
LGTM

**Comment by XuHuaiyu** [comment]
LGTM

**Comment by ngaut** [comment]
Please merge the latest master.

File: kv/kv.go [hunk 1]
```diff
@@ -187,7 +186,7 @@ type Storage interface {
 	GetClient() Client
 	// Close store
 	Close() error
-	// Storage's unique ID
+	// UUID return a unique ID which represents a Storage.
```
Comment (zimulala) [inline kv/kv.go]: s/return/returns

**Pull request merged**